modules/monitoring/application/forms/Command/Object/CheckNowCommandForm.php

```diff
@@ -21,6 +21,26 @@ public function init()
         $this->setAttrib('class', 'inline');
     }
 
+    /**
+     * SubmitButton Label
+     *
+     * @var SubmitButtonLabel
+     */
+    protected $submitButtonLabel = 'Check now';
+
+    /**
+     * Set the submit button label
+     *
+     * @param   String $submitButtonLabel
+     *
+     * @return  $this
+     */
+    public function setSubmitButtonLabel(String $submitButtonLabel)
```

Comment:
The type hint for strings does only work for PHP version 7+. Since we still support 5.6+ this must be removed.